pyproject.toml

```
@@ -29,7 +29,7 @@ dependencies = [
     "fpdf2",
     "jsonschema",
     "psutil",
-    "pyedb>=0.4.0,<0.11",
+    "pyedb>=0.4.0,<0.12",
```

```suggestion
    "pyedb>=0.4.0",
```
In fact, we might just removed the upper bound since we won't be able to test the new release of pyedb through dependabot. Indeed, we would already be allowing this version.
Moreover, `pyedb` release process tests that the current pyedb dev version does not fail pyaedt units tests.